internal/services/desktopvirtualization/virtual_desktop_host_pool_registration_info_resource.go

```diff
@@ -0,0 +1,202 @@
+package desktopvirtualization
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/services/preview/desktopvirtualization/mgmt/2021-09-03-preview/desktopvirtualization"
+	"github.com/Azure/go-autorest/autorest/date"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/clients"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/locks"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/services/desktopvirtualization/parse"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/pluginsdk"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/tf/validation"
+	"github.com/hashicorp/terraform-provider-azurerm/internal/timeouts"
+	"github.com/hashicorp/terraform-provider-azurerm/utils"
+)
+
+func resourceVirtualDesktopHostPoolRegistrationInfo() *pluginsdk.Resource {
+	return &pluginsdk.Resource{
+		Create: resourceVirtualDesktopHostPoolRegistrationInfoCreateUpdate,
+		Read:   resourceVirtualDesktopHostPoolRegistrationInfoRead,
+		Update: resourceVirtualDesktopHostPoolRegistrationInfoCreateUpdate,
+		Delete: resourceVirtualDesktopHostPoolRegistrationInfoDelete,
+
+		Timeouts: &pluginsdk.ResourceTimeout{
+			Create: pluginsdk.DefaultTimeout(60 * time.Minute),
+			Read:   pluginsdk.DefaultTimeout(5 * time.Minute),
+			Update: pluginsdk.DefaultTimeout(60 * time.Minute),
+			Delete: pluginsdk.DefaultTimeout(60 * time.Minute),
```

Comment:
presuming this has been copied & paste from the resource:

```suggestion
			Create: pluginsdk.DefaultTimeout(30 * time.Minute),
			Read:   pluginsdk.DefaultTimeout(5 * time.Minute),
			Update: pluginsdk.DefaultTimeout(30 * time.Minute),
			Delete: pluginsdk.DefaultTimeout(30 * time.Minute),
```